### PR TITLE
Lift the embargo!

### LINKED ISSUE #188: announcement post
embargoed until 9 am Pacific Time on Tuesday, August 31, 2021. 

- [ ] blog
- [ ] tweet

### PR DESCRIPTION
**Pull request by IAlibay**
Fixes #188

### CODE REVIEW

**Review by orbeckst — COMMENTED** [review]
lgtm

File: _posts/2021-08-31-CZI-EOSS4.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+---
+layout: post
+title: Chan Zuckerberg Initiative EOSS4 Award
+---
+
+MDAnalysis is excited to announce that we have been awarded an [Essential Open Source Software for Science][] (EOSS) grant by the Chan Zuckerberg Initiative for our proposal [“MDAnalysis: Faster, extensible molecular analysis for reproducible science”][].
+
+Over the next two years, this award will fund work on two major aims that will improve MDAnalysis for our users and make it easier for scientists to contribute code, with a view towards increasing code reuse and interoperability and increasing reproducibility:
```
Comment (lilyminium) [inline _posts/2021-08-31-CZI-EOSS4.md:8]:
```suggestion
Over the next two years, this award will fund work on two major aims that will improve MDAnalysis for our users and make it easier for scientists to contribute code, with a view towards increasing code reuse, interoperability, and reproducibility:
```

**Comment by orbeckst** [comment]
This is weird — I clicked "approve" but it only shows up as a comment.

File: _posts/2021-08-31-CZI-EOSS4.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
...Toolkits (“MDAKit”) ecosystem. MDAKits are inspired by SciPy’s SciKits. An MDAKit will contain code based on MDAnalysis that solves specific scientific problems. An MDAKit can be written by anyone and hosted anywhere. In order to streamline the process we will provide a cookiecutter-based template repository. MDAKits will have a well-defined API, have documentation, will be registered with MDAnalysis, and will be continuously tested  for compatibility with the current version of MDAnalysis. As part of the registration process, code will be reviewed by MDAnalysis developers and we will encourage the scientific publication of the package, e.g., in the [Journal of Open Source Software][]. We will also refactor a number of analysis modules as separate MDAKits. Overall, the introduction of MDAKits will enable the rapid development and dissemination of new components and tools that can be released independently of the main package. We will soon describe the details of MDAKits in a white paper and seek a discussion with the wider community. Ultimately, MDAKits should help address two long-standing challenges in the biomolecular simulation community: effort duplication and reproducibility.
```
Comment (lilyminium) [inline _posts/2021-08-31-CZI-EOSS4.md:18]:
```suggestion
We will launch the MDAnalysis Toolkits (“MDAKit”) ecosystem. Inspired by SciPy’s SciKits, an MDAKit will contain code based on MDAnalysis that solves specific scientific problems. An MDAKit can be written by anyone and hosted anywhere. In order to streamline the process we will provide a cookiecutter-based template repository. MDAKits will have a well-defined API, have documentation, will be registered with MDAnalysis, and will be continuously tested for compatibility with the current version of MDAnalysis. As part of the registration process, code will be reviewed by MDAnalysis developers and we will encourage the scientific publication of the package, e.g., in the [Journal of Open Source Software][]. We will also refactor a number of analysis modules as separate MDAKits. Overall, the introduction of MDAKits will enable the rapid development and dissemination of new components and tools that can be released independently of the main package. We will soon describe the details of MDAKits in a white paper and seek a discussion with the wider community. Ultimately, MDAKits should help address two long-standing challenges in the biomolecular simulation community: effort duplication and reproducibility.
```

Comment (IAlibay) [reply _posts/2021-08-31-CZI-EOSS4.md:18]: How much of a re-write is this? We already have approved text...

Comment (IAlibay) [reply _posts/2021-08-31-CZI-EOSS4.md:18]: Tagging @orbeckst

**Review by lilyminium — APPROVED** [review]
Thanks @IAlibay! Have you checked if this builds properly locally or should I?

**Comment by IAlibay** [comment]
> Thanks @IAlibay! Have you checked if this builds properly locally or should I?

Not yet - if you could check that'd be great (otherwise we can do it live 🙃 )

File: _posts/2021-08-31-CZI-EOSS4.md (hunk 2 above)
Comment (lilyminium) [reply _posts/2021-08-31-CZI-EOSS4.md:18]: Sorry, it's just grammar.

"MDAKits are inspired by SciPy’s SciKits. An MDAKit will contain code based on MDAnalysis that solves specific scientific problems." -> " Inspired by SciPy’s SciKits, an MDAKit will contain code based on MDAnalysis that solves specific scientific problems."

and I deleted an extra space

**Comment by lilyminium** [comment]
Screen Shot 2021-08-31 at 9 14 43 am

Builds goodly!

We might need to update our local instructions -- at least I had to `bundle add webrick` before things worked.

File: _posts/2021-08-31-CZI-EOSS4.md (hunk 2 above)
Comment (orbeckst) [reply _posts/2021-08-31-CZI-EOSS4.md:18]: will merge

**Review by orbeckst — APPROVED** [review]
release the kraken

**Pull request merged**